### PR TITLE
Nested resources

### PR DESCRIPTION
**Pull request by OriMelamed**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,8 +19,12 @@ gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.7'
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 4.0'
-# Use Active Model has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
+
+#Use Active Model has_secure_password
+gem 'bcrypt', '~> 3.1.7'
+
+gem 'will_paginate', '~> 3.3'
```
Comment (dpaluy) [inline Gemfile:27]: @OriMelamed This gem is very inefficient and we should use https://github.com/ddnexus/pagy instead

File: app/controllers/buyers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+    def show
+        @buyer = Buyer.find(params[:id])
+    end
+
+    def index
+        @buyers = Buyer.all
+    end
+
+end
```
Comment (dpaluy) [inline app/controllers/buyers_controller.rb]: END of line is missing, please use Visual Studio plugin to automate this

**Review by dpaluy — CHANGES_REQUESTED** [review]

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root 'pages#home'
   get 'about', to: 'pages#about'
-  resources :invoices
+  resources :buyers do
```
Comment (dpaluy) [inline config/routes.rb]: This looks good. I recommend checking `:shallow`

**Review by dpaluy — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (OriMelamed) [reply Gemfile:27]: change to pagy for the pagination

**Pull request merged**